ait/core/server/handlers/packet_handler.py

```diff
@@ -19,27 +19,48 @@ def __init__(self, input_type=None, output_type=None, **kwargs):
                            If packet is specified but not present in default tlm dict.
         """
         super(PacketHandler, self).__init__(input_type, output_type)
-        self.packet = kwargs.get("packet", None)
+        self.packet_type = kwargs.get("packet", None)
+        self.tlm_dict = tlm.getDefaultDict()
 
-        if not self.packet:
-            msg = 'PacketHandler: No packet name provided in handler config as key "packet"'
+        if not self.packet_type:
+            msg = f'PacketHandler: No packet type provided in handler config as key "packet"'
             raise ValueError(msg)
 
-        tlm_dict = tlm.getDefaultDict()
-        if self.packet not in tlm_dict:
-            msg = "PacketHandler: Packet name {} not present in telemetry dictionary".format(
-                self.packet
-            )
-            msg += " Available packet types are {}".format(tlm_dict.keys())
+        if self.packet_type not in self.tlm_dict:
+            msg = f"PacketHandler: Packet name '{self.packet_type}' not present in telemetry dictionary."
+            msg += f" Available packet types are {self.tlm_dict.keys()}"
             raise ValueError(msg)
 
-        self._pkt_defn = tlm_dict[self.packet]
+        self._pkt_defn = self.tlm_dict[self.packet_type]
```

Comment:
@JHofman728 Since you are already grabbing the packet definition here, there really should be no need for the new `get_packet_lengths()` function below.